cards.js

```diff
@@ -116,6 +116,30 @@ class CardSet {
     }
   }
 
+  includes(card) {
+    let result = false;
+    this.cards.forEach(x => {
+      if(x.rank == card.rank && x.suit == card.suit){
+        result = true;
+      }
```

Comment:
- Neefektīvi izmantot forEach, jo ja kārti atrod jau agri, tad tik un tā turpina iet cauri visam setam. Ja ir situācijas, kur var būt nepieciešams izlekt ārā no iteratora, tad pareizāk tomēr lietot for (value of array) ciklu.
- Protams, te jāizmanto plānotā Card.equals metode.

Reply:
So es jau aizvietoju ar cards.filter(jo es uzticos ka js metodes ir labaak optimizeetas nekaa kods ko es rakstu)

Reply:
Bet buutu jauki pagaidiit liidz pievieno Card.equals, jaa.

Reply:
Tagad izmanto compareCards funckiju ko @sakars izveidoja, tikai nez kapeec ir ieksaa CukasGame, so man vajag nedaudz nesmuku atrisinaajumu veikt.

oh well